fuel-core/src/cli/run/p2p.rs

```diff
@@ -147,17 +155,26 @@ impl From<P2pArgs> for anyhow::Result<P2PConfig> {
             }
         };
 
+        // Reserved nodes do not count against the configured peer input/output limits.
+        let reserved_nodes_count = args.reserved_nodes.len();
+
         let gossipsub_config = default_gossipsub_builder()
-            .mesh_n(args.ideal_mesh_size)
-            .mesh_n_low(args.min_mesh_size)
-            .mesh_n_high(args.max_mesh_size)
+            .mesh_n(args.ideal_mesh_size + reserved_nodes_count)
```

Comment:
How do we prevent non-reserved peers from using up the extra connection slots added by `reserved_nodes_count`? Will this be addressed in a follow-up PR to ensure we continuously attempting to connect to reserved peer?

Reply:
yes that will be done with the incoming PR that will handle Peer Connections of reserved and non-reserved nodes